easybuild/framework/easyconfig/tweak.py

```diff
@@ -807,6 +914,16 @@ def map_easyconfig_to_target_tc_hierarchy(ec_spec, toolchain_mapping, targetdir=
     """
     # Fully parse the original easyconfig
     parsed_ec = process_easyconfig(ec_spec, validate=False)[0]
+
+    # There are some common versionsuffixes (like '-Python-(%pyver)s') that also need dynamic searching/updating
+    versonsuffix_mapping = map_common_versionsuffixes('Python', '2', parsed_ec['ec']['toolchain'], toolchain_mapping)
+    versonsuffix_mapping.update(map_common_versionsuffixes('Python', '3', parsed_ec['ec']['toolchain'],
```

Comment:
Hmm, can we avoid hardcoding versions here somehow? What if there's a Python 4.x at some point?